modules/templates/proc-adding-templates.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-templates_{context}"]
+= Importing an existing template to {product}
+
+You can add an existing template to your {product} instance by using the Catalog Processor.
+
+.Prerequisites
+
+* You have created a directory or repository that contains at least one template YAML file.
+
+.Procedure
+
+. In the `app-config.yaml` configuration file, add the `catalog.configuration` section:
++
+[source,yaml]
+----
+# ...
+catalog:
+  locations:
+    - type: url # <1>
+      target: https://<repository_url>/templates # <2>
+    - type: url
+      target: https://<repository_url>/example-template.yaml
+   - type: file # <3>
+      target: ./templates # <4>
+   - type: file
+      target: ./templates/example-template.yaml
+# ...
+----
+<1> If you are importing templates from a repository, such as GitHub or GitLab, use the `url` type.
+<2> Specify the URL for either a directory where multiple templates are stored, or an individual template as a YAML file.
+<3> If you are importing templates from a local directory, use the `file` type.
+<4> Specify the path to the local directory that contains your template YAML files, or an individual template as a YAML file.
+
+. {product} automatically detects new templates that are added to a configured location, including templates that are added to a configured repository directory, and makes them available in the catalog.
+
```

Comment:
You'll also need to discuss `Catalog Rules` (https://backstage.io/docs/features/software-catalog/configuration#catalog-rules)

By default `Template` is NOT included in the rules so even if you point to a template type backstage won't ingest it unless you add the rule to allow the `Template` type

Reply:
Yes we should explain how rules apply to templates otherwise you won't see them like also if you don't provide a github token having access to a git repository (= type: url) then backstage will not be able to read and load them